### DISCUSSION TITLE
While logging into postgres, getting the below error

### DISCUSSION THREAD

**Discussion by zombierider0000** [outline/outline#7599] [General] [answered]
psql: error: connection to server at "localhost" (::1), port 5432 failed: FATAL: password authentication failed for user "POSTGRES"

**Comment by Syjgin** [comment]
Looks like self-hosted installation guide is lacks section about postgres new user and password addition. I can't find such infirmation, so command 
`docker compose run --rm outline yarn db:create --env=production-ssl-disabled `
returns connection error. 
Docker installation guide is unusable now

**Reply by tommoor** [reply]
The guide includes this information in the example docker compose environment variables:

https://docs.getoutline.com/s/hosting/doc/docker-7pfeLP5a8t

**Reply by Syjgin** [reply]
`docker compose run --rm outline yarn db:create --env=production-ssl-disabled
WARN[0000] /root/getoutline/docker-compose.yml: the attribute `version` is obsolete, it will be ignored, please remove it to avoid potential confusion 
[+] Creating 5/0
 ✔ Network getoutline_default         Created                                                            0.1s 
 ✔ Volume "getoutline_database-data"  Created                                                            0.0s 
 ✔ Volume "getoutline_storage-data"   Created                                                            0.0s 
 ✔ Container getoutline-postgres-1    Created                                                            0.0s 
 ✔ Container getoutline-redis-1       Created                                                            0.0s 
[+] Running 2/2
 ✔ Container getoutline-postgres-1  Started                                                              0.2s 
 ✔ Container getoutline-redis-1     Started                                                              0.2s 
yarn run v1.22.22
$ sequelize db:create --env=production-ssl-disabled

Sequelize CLI [Node: 20.17.0, CLI: 6.6.2, ORM: 6.37.3]

Loaded configuration file "server/config/database.json".
Using environment "production-ssl-disabled".

ERROR: connect ECONNREFUSED 127.0.0.1:5432

error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
root@zealous-plot:~/getoutline# docker compose run --rm outline yarn db:migrate --env=production-ssl-disabled
^[[DWARN[0000] /root/getoutline/docker-compose.yml: the attribute `version` is obsolete, it will be ignored, please remove it to avoid potential confusion 
[+] Creating 2/0
 ✔ Container getoutline-redis-1     Running                                                              0.0s 
 ✔ Container getoutline-postgres-1  Running                                                              0.0s 
yarn run v1.22.22
$ sequelize db:migrate --env=production-ssl-disabled

Sequelize CLI [Node: 20.17.0, CLI: 6.6.2, ORM: 6.37.3]

Loaded configuration file "server/config/database.json".
Using environment "production-ssl-disabled".

ERROR: connect ECONNREFUSED 127.0.0.1:5432

error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
`

Unable to connect to the database with example username & password. No information about creating required postrgres user inside container is supplied. So this guide can't be used to create working self-hosted solution

**Reply by tommoor** [reply, marked as answer]
> No information about creating required postrgres user inside container is supplied.

It happens automatically based on the environment variables supplied. Read the Postgres docker image documentation for more details on that front

**Reply by Syjgin** [reply]
But the documentation does not link to this guide and does not even mention that nothing will work without creating the right user. If you install Affine in Docker, for example, after applying the commands from the guide, you will get a working instance. In the case of the Getoutline installation guide, this will not happen
